src/components/services/Services.vue

```diff
@@ -278,9 +304,24 @@
         );
         this.displayConfirmationDialog = false;
       },
+      onTransfer() {
+        this.$store.dispatch(A.TRANSFER_SERVICES);
```

Comment:
No need to dispatch an action if the state is not changed in vuex state management. A websocket notification for the backend to perform the action would suffice